### ISSUE TITLE
Mod update

### ISSUE THREAD

**Issue by MOOOOs** [Ferrster/Snowrunner-Manual-Gearbox-Mod#34]
Mod may need to be updated again because useless pros update. It works fine in normal gameplay but will crash in proving grounds when trying to use free camera bound on num controls same as gears it didn't happen before update.

**Comment by Ferrster** [comment]
Just tested it and it works fine on my Steam version of the game. Function offsets are all correct and nothing is crashing, even when bind gear from 1 to 8 on numpad. I'm not sure mod is the problem. But if crashes persist, the short video will be helpful.

**Comment by MOOOOs** [comment]
have you tried to toggle free cam and move it?

**Comment by Ferrster** [comment]
Yes, I did move free cam with num keys, and it didn't crash. I honestly have no idea why free cam would cause this.
Is it specific num key that causes the crash or is it random?
